R/cache.R

```diff
@@ -317,15 +325,21 @@ build_cached_function <- function(cached_function) {
     }
 
     ## Check whether **force.** was set
-    if (isTRUE(is_force)) {
+    if (is_force) {
       force. <- TRUE
       message("`force.` detected. Overwriting cache...\n")
     } else force. <- FALSE
 
-    cachemeifyoucan:::execute(
-      cachemeifyoucan:::cached_function_call(`_uncached_function`, call,
+    fcn_call <- cachemeifyoucan:::cached_function_call(`_uncached_function`, call,
         parent.frame(), tbl_name, `_key`, `_con`, force., `_batch_size`)
-    )
+
+    ## Grab the all keys
+    keys <- fcn_call$call[[fcn_call$key]]
+
+    ## Log cache metadata if in debug mode
+    status <- debug_info(fcn_call, keys)
```

Comment:
`options(cachemeifyoucan.debug = TRUE)` should continue to print the debug info even if `dry. = FALSE`.